web_favicon/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Therp BV <http://therp.nl>
+# Copyright 2016 Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Custom shortcut icon",
+    "version": "12.0.1.0.0",
+    "author": "Therp BV, "
```

Comment:
@tarteo Add website as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

Reply:
Added it. Thanks :)